url/urlsearchparams-append.any.js

```diff
@@ -1,11 +1,7 @@
-<!doctype html>
-<html>
-<head>
-<meta charset="utf8">
-<link rel="help" href="http://url.spec.whatwg.org/#dom-urlsearchparams-append">
-<script src="/resources/testharness.js"></script>
-<script src="/resources/testharnessreport.js"></script>
-<script>
+/*
+    https://url.spec.whatwg.org/#dom-urlsearchparams-append
+ */
```

Comment:
No need to keep this. Or any of the links. It's pretty self-explanatory where this is all documented.

Reply:
Thanks for the feedback @annevk!

I'm curious to hear why you think its obvious and self-explanatory. I agree it might be the case for spec authors and implementors. However, for new WPT contributors without a strong standards background, I think this link would be pretty helpful.

Reply:
I was thinking the README had a link to the standard (as a standalone section), but it seems like it doesn't. I think we should add that. Test metadata tends to go out-of-date.

Reply:
Sounds good @annevk. Will update.